datagenerator/Closures.py

```diff
@@ -863,7 +863,7 @@ def find_top_lith_horizons(self):
         self.top_lith_indices = np.array(top_lith_indices)
         self.top_lith_facies = self.facies[top_lith_indices]
 
-        # return top_lith_indices
```

Comment:
i was attempting to get the horizon tops by returning top lithology indices. we can ignore this change